src/cmds/scripts/pbs_habitat.in

```diff
@@ -521,6 +521,22 @@ upgrade_pbs_schema() {
 		return 1
 	fi
 
+
+	if [ "$ver" = "1.2.0" ]; then
+		upgrade_pbs_schema_from_v1_2_0
+		ret=$?
+		if [ $ret -ne 0 ]; then
+			return $ret
+		fi
```

Comment:
First release of Open Source PBSPro has schema version 1.2. No support for older schema version.